package.json

```diff
@@ -215,6 +215,7 @@
     "eslint": "^6.8.0",
     "eslint-config-satya164": "^3.1.6",
     "eslint-import-resolver-babel-module": "^5.1.2",
+    "graphql": "^15.3.0",
```

Comment:
Where are we using it?

Reply:
It was previously resolved as subdep but now it's somewhere needed explicitly. Crashed w/o this